src/Mod/Path/PathScripts/PathPropertyBagGui.py

```diff
@@ -22,12 +22,13 @@
 
 import FreeCAD
 import FreeCADGui
-import PathGui
+#import PathGui
```

Comment:
This needs to stay, all Gui implementations should import this - it ensures that the ViewProvider works even when the Path WB hasn't been loaded yet.